### PR TITLE
NR89215 Sendmail - new quickstart

### PR DESCRIPTION
**Pull request by dikshab0202**
Ticket : [89215](https://issues.newrelic.com/browse/NR-89215)
Description: Created new quickstart - Sendmail
Dashboard url : https://onenr.io/0nQxPZW7AQV

# Summary

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

_The owners of this repo are not experts in the subject matter of the quickstarts. We review for the quickstart to be functional and for security risks. If you are seeking feedback on the content of the quickstart, please seek out a subject matter expert. If you are not an internal NR contributor, we can do our best to connect you with a content reviewer._

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you include a Data source and Documentation reference?
- [ ] Are all documentation links publicly accessible?
- [ ] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [ ] Did you check your descriptive content for spelling and grammar errors?
- [ ] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [ ] Does the PR contain a screenshot for each of your dashboards?
- [ ] Do your screenshots show data?
- [ ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### CODE REVIEW

**Review by mdumpati — APPROVED** [review]
LGTM

File: alert-policies/sendmail/MessageUndelivered.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+# Description and details
+description: |+
+  This alert is triggered when 5 percent or more of messages are undelivered.
+# Type of alert
+type: STATIC
+
+# NRQL query
+nrql:
+  query: "SELECT 100 - ((latest(SendmailMessageAcceptedForDelivery.value) * 100) / latest(SendmailMessageStatus.sent)) as '(%) message undelivered' from SendmailMessageStatus, SendmailMessageAcceptedForDelivery"
```
Comment (mdumpati) [inline alert-policies/sendmail/MessageUndelivered.yml:12]:
```suggestion
  query: "SELECT ((latest(SendmailMessageAcceptedForDelivery.value)) / latest(SendmailMessageStatus.sent)) * 100 as '(%) message undelivered' from SendmailMessageStatus, SendmailMessageAcceptedForDelivery"
```

**Review by mdumpati — COMMENTED** [review]
@dikshab0202 Please check my comments

**Review by mdumpati — APPROVED** [review]
LGTM

**Review by seemantk — APPROVED** [review]

**Pull request merged**